docs/pages/guides/using-custom-fonts.md

```diff
@@ -112,19 +113,19 @@ If you need to use another format, you may need to [customize the Metro bundler
 
 But, for reference, here is a table of which formats work on which platforms.
 
-| Format | Web | iOS | Android |
-| ------ | --- | --- | ------- |
-| ttf    | ✅  | ✅  | ✅      |
-| otf    | ✅  | ✅  | ✅      |
-| woff   | ✅  | ✅  |         |
-| woff2  | ✅  | ✅  |         |
-| dfont  |     |     | ✅      |
-| svg    | ✳️  |     |         |
-| eot    | ✳️  |     |         |
-| fon    |     |     |         |
-| bdf    |     |     |         |
-| ps     |     |     |         |
-| ttc    |     |     |         |
+| Format | Web         | iOS         | Android     |
```

Comment:
What do you think about filling empty cells with `NoIcon`?

Reply:
I was wondering if that will be a good way to go. Thanks for the heads up. I'll make that change.

Reply:
Also it would be nice to reorder the columns in the alphabetical order.